flag/standard/hekla/map.xml

```diff
@@ -1,5 +1,4 @@
-<?xml version="1.0"?>
-<map proto="1.4.0">
+<map proto="1.4.2">
 <name>Hekla</name>
 <version>1.0.3</version>
```

Comment:
This should be bumped, probably just missed in merge from main after Pugzy updated it